relay-server/src/services/store.rs

```diff
@@ -489,21 +499,9 @@ impl StoreService {
         scoping: Scoping,
         start_time: Instant,
         remote_addr: Option<String>,
-        attachments: Vec<ChunkedAttachment>,
+        individual_attachments: Vec<ChunkedAttachment>,
+        inline_attachments: Vec<ChunkedAttachment>,
```

Comment:
This function has become pretty trivial as you noted in the PR description. Feel free to factor it out if you prefer.